source/jormungandr/jormungandr/instance.py

```diff
@@ -587,6 +587,48 @@ def min_ridesharing(self):
         instance_db = self.get_models()
         return get_value_or_default('min_ridesharing', instance_db, self.name)
 
+    @property
+    def max_bike_direct_path_distance(self):
+        # type: () -> int
+        instance_db = self.get_models()
+        return get_value_or_default('max_bike_direct_path_distance', instance_db, self.name)
```

Comment:
you can simplify your codes by using `_make_property_getter`

```suggestion
    max_bike_direct_path_distance = _make_property_getter('max_bike_direct_path_distance')
```